### DISCUSSION TITLE
About Statistical Power and P-Value Validity with Small Replicate Sizes

### DISCUSSION THREAD

**Discussion by helloworld1631** [Nesvilab/FragPipe-Analyst-doc#30] [Q&A] [answered]
Hi, 

I've been utilizing Fragpipe-Analyst for differential expression analysis in my experiment, where each condition has two replicates. After uploading the quantification results, I found that the software still generates robust differential expression analysis and plotting.

From your pre-print manuscript, I understand that Fragpipe-Analyst employs feature-wise linear models along with empirical Bayesian statistics from Limma to identify statistically significant differences in abundance between conditions. Given that my experiments involve a small number of replicates (only two per condition), I'm concerned about the reliability of the p-values or adjusted p-values produced. Could you provide some insights on whether these differential expression results are still trustable and if they offer sufficient statistical power with such limited replicates?

Thank you!

**Comment by hsiaoyi0504** [comment, marked as answer]
Hi @helloworld1631, I feel the question is a little beyond the scope of FragPipe-Analyst.
One of several objectives of FragPipe-Analyst is to help users correctly take FragPipe's result to downstream analysis including the statistical procedure like differential expression analysis mentioned.

However, that doesn't mean FragPipe-Analyst magically boost the performance of several methods such as Limma used here. Your concern is reasonable, but it's not about reliability of the p-values or adjusted p-values. I think Limma has fair enough theoretical p-values or adjusted p-values calculation procedure and reliable, but given you have a small sample size, there is not much we can do. If you are stating a biological effects in your experiments, you will need more samples or more evidences from other approaches (no the same bioinformatics analysis discussed here).

**Reply by helloworld1631** [reply]
I see. I noticed that the FragPipe-Analyst software does not provide specific restrictions or suggestions for replicate settings, so that is why I wanted to ask.

Thanks for your explanation!
